charts/pega/templates/_pega-service.tpl

```diff
@@ -16,12 +17,15 @@ metadata:
     traefik.ingress.kubernetes.io/max-conn-amount: '10'
     # Manually set the cookie name for sticky sessions
     traefik.ingress.kubernetes.io/session-cookie-name: UNIQUE-PEGA-COOKIE-NAME
```

Comment:
Does this need to be a values.yaml parameter?  Maybe default the value to the deployment-tier name?